### ISSUE TITLE
Failed to build release

### ISSUE THREAD

**Issue by kesha-antonov** [gre/gl-react#106]
Hey @gre!

When I "Archieve" my project through XCode it throws error: ```'React/RCTJSCExecutor.h' file not found```

#### library version:

```bash
├── gl-react@3.3.0 
├── gl-react-native@3.3.0 
└── UNMET PEER DEPENDENCY react@16.0.0-alpha.12
```

#### Expected behavior

Build succeffully.

#### Actual behavior

Throws error.

#### Steps to reproduce the behavior

1. Set project to "release"
2. XCode -> Product -> Archieve.

**Comment by kesha-antonov** [comment]
<img width="299" alt="screen shot 2017-05-08 at 18 24 27" src="https://cloud.githubusercontent.com/assets/11584712/25814364/e9cfe79e-3425-11e7-95aa-eda93966e924.png">
<img width="1121" alt="screen shot 2017-05-08 at 19 37 51" src="https://cloud.githubusercontent.com/assets/11584712/25814371/efeaa79a-3425-11e7-8bc6-1c565a5272da.png">

**Comment by kesha-antonov** [comment]
https://github.com/gre/gl-react/blob/master/packages/gl-react-native/ios/EXGLView.m

**Comment by kesha-antonov** [comment]
This works:
```
#import "EXGLView.h"

#include <OpenGLES/ES2/gl.h>
#include <OpenGLES/ES2/glext.h>

#import <React/RCTUtils.h>
// #import <React/RCTJSCExecutor.h>
#import "../../react-native/React/Executors/RCTJSCExecutor.h"

#import "EXUnversioned.h"
// #import "EXGL.h"
#import "../cpp/EXGL.h"
```

Am I missing something?
I'm new to C-world.

Or maybe paths changed in RN 0.44

**Comment by gre** [comment]
maybe it's not this but can you try to see in your project if you have this flag? 

<img width="445" alt="screen shot 2017-05-08 at 21 25 47" src="https://cloud.githubusercontent.com/assets/211411/25821329/f77922d4-3434-11e7-93ca-a22672501599.png">

**Comment by kesha-antonov** [comment]
<img width="940" alt="screen shot 2017-05-08 at 22 39 12" src="https://cloud.githubusercontent.com/assets/11584712/25821833/46628bec-343f-11e7-82bb-5c253f6515db.png">

**Comment by gre** [comment]
did that happen specifically after a recent upgrade of RN ? you used to be able to build gl-react before?

**Comment by kesha-antonov** [comment]
Yes.
But I use master of RN. Maybe it changes something

Anyway it builds now. And no one reporting issues. So it's a low priority

**Comment by kelset** [comment]
Had the same issue, the workaround proposed by @kesha-antonov works. But I think this issue may be related to the "original version" of the project. In my test project build fresh from RN 0.44 didn't have this issue, but when I tried adding `gl-react` to a project that was probably first created on a RN version lower than 40 (then updated ofc) I had this issue.

**Comment by gre** [comment]
sorry for the delay, still trying to actually run things..
phew, i'm having an hard time running the repository projects these days, I constantly get issues with yarn registry...
I guess i'm going to have to make a single project where i manually copy paste the lib.

going to take a closer look this weekend

**Comment by kelset** [comment]
`3.7.0` fixes it for me (RN 0.45) - @kesha-antonov can you try with the new version?

**Comment by gre** [comment]
(we'll reopen if it's not fixed)
